### PR TITLE
dilithium no longer sucks ass and never gets used

### PR DESCRIPTION
**Pull request by athena413**
this should make dilithium gas more used and better overall

#### Changelog

:cl:  
tweak: dilithium obtained from dilithium is now times ten
tweak: dilithium asymptote lowered to 425 from 4000
/:cl:

### CODE REVIEW

File: code/__DEFINES/~yogs_defines/reactions.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //Cold Fusion
-#define FUSION_TEMPERATURE_THRESHOLD_MINIMUM 4000 // This is the minimum possible required temperature that dilithium can lower fusion to.
+#define FUSION_TEMPERATURE_THRESHOLD_MINIMUM 425 // This is the minimum possible required temperature that dilithium can lower fusion to.
 #define DILITHIUM_LAMBDA 0.0087 // Affects how much Dilithium is required to get the required fusion temperature down to FUSION_TEMPERATURE_THRESHOLD_MINIMUM
```
Comment (jcatjmeow) [inline code/__DEFINES/~yogs_defines/reactions.dm]: Take these //comments and make them /// and put them one line above the defines

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

File: code/__DEFINES/~yogs_defines/reactions.dm (hunk 1 above)
Comment (athena413) [reply code/__DEFINES/~yogs_defines/reactions.dm]: done, also autodocced the BZ stuff

**Comment by Moja10** [comment]
Diulithem is two lithem,

**Review by fluffe9911 — APPROVED** [review]
p o w e r c r e e p

**Pull request merged**